utils/markdown/index.ts

```diff
@@ -18,12 +22,15 @@ export function format(text: string, options = {}, emojiMap: EmojiMap) {
 }
 
 export function formatWithRenderer(text: string, renderer: marked.Renderer) {
+    const config = getConfig(store.getState());
+
     const markdownOptions = {
         renderer,
         sanitize: true,
         gfm: true,
         tables: true,
         mangle: false,
+        inlinelatex: config.EnableInlineLatex === 'true',
```

Comment:
@KobeBergmans Actually, the test server we set up isn't picking up the correct server branch, so it doesn't know the new setting exists. To get around that, could you temporarily set this to true so we can try it out?

Reply:
Yes i'll do it right away!

Reply:
@hmhealey the test server now works as intended.